scout/adapter/mongo/panel.py

```diff
@@ -512,22 +501,23 @@ def apply_pending(self, panel_obj, version):
 
             current_update = updates[hgnc_id]
             action = current_update["action"]
-            info = current_update["info"]
 
             # If action is delete we do not add the gene to new genes
             if action == "delete":
                 continue
 
             if action == "edit":
-                for field in info_fields:
-                    if field in info:
-                        gene[field] = info[field]
+                # reset all fields except hgnc and symbol:
+                for key in EXPORT_PANEL_FIELDS[2:]:
```

Comment:
This should fix the resetting values issue

Reply:
👍 Looks good; we trust the update info to have and set any true values and the rest are unset.